core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.spec.ts

```diff
@@ -265,6 +265,41 @@ describe('DotExperimentsConfigurationVariantsComponent', () => {
             expect(output).toEqual({ ...variants[1], name: newVariantName });
         });
 
+        it('should save when press enter', () => {
+            spyOn(spectator.component, 'editVariantName');
+
+            spectator.component.vm$ = of({
+                status: {
+                    status: Status.IDLE,
+                    isOpen: false,
+                    experimentStep: ExperimentSteps.GOAL
+                }
+            });
+
+            const variants: Variant[] = [
+                { id: '1', name: DEFAULT_VARIANT_NAME, weight: '50.00', url: 'url' },
+                { id: '2', name: 'to edit', weight: '50.00', url: 'url' }
+            ];
+
+            spectator.setInput({
+                variants
+            });
+
+            spectator.query(Inplace).activate();
+
+            spectator.detectComponentChanges();
+
+            const inplaceInput = spectator.query(byTestId('inplace-input')) as HTMLInputElement;
+            inplaceInput.value = 'new value';
+            spectator.detectComponentChanges();
+
+            spectator.dispatchKeyboardEvent(inplaceInput, 'keydown', 'Enter');
+
+            spectator.detectComponentChanges();
+
+            expect(spectator.component.editVariantName).toHaveBeenCalled();
```

Comment:
`toHaveBeenCalledWith` to check the params ?

Reply:
Used toHaveBeenCalledWith.